apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/style.scss

```diff
@@ -1,10 +1,16 @@
-.block-editor .wp-block-a8c-site-description {
-	&, &:focus {
-		display: inline;
-		color: #767676;
-		font-size: 1.125em;
-		font-weight: normal;
-		letter-spacing: -0.01em;
-		margin: 0;
+.block-editor {
+	.wp-block-a8c-site-description {
+		&,&:focus {
```

Comment:
Non-blocking, but careful not to nest too deeply. Use your best judgement, but over time this gets harder to read if it's not viewable all at once in the code editor.

Reply:
yep, good point, this is about as deep as I would normally nest.